

bool FAULT_EFI          # True : good; False: disconnect between MS3 vs FLEX
bool FAULT_MS3          # True : good; False: error Laserscan
bool FAULT_GENT         # True : good; False: disconnect between IPC vs FLEX
bool FAULT_EDM          # True : good; False: Contactor motor error
bool FAULT_DRIVER       # 
bool FAULT_ENC

